### PR TITLE
Added two new e2e tests

### PR DESCRIPTION
**Pull request by raya311**
# Pull Request

## Summary

Added two new E2E tests.
1. Toggle packed/unpacked items
2. Delete a trip

## Why

To increase the test coverage

## How to Verify

'npx playwright test' should pass all the tests

## Checklist (Definition of Done)

- Branch follows team naming pattern (`week<NUM>/<short-description>`)
- PR opened early and kept reviewable in size
- At least one teammate review requested
- CI checks are passing
- Lint passes locally (`npm run eslint`)
- Tests pass locally (`npm run test`)
- New behavior includes starter tests or test plan notes
- Documentation updated if needed (`README`, `docs/`, ADRs)

## Notes for Reviewers

Anything specific you'd like feedback on?
- Edge cases?
- Architecture concerns?
- Naming?
- Test coverage?

### CODE REVIEW

File: docs/beta/week12-quality.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+**Impact:** Validates end-to-end workflows on real browsers, catching UI/API integration issues.
+
+---
+
+## Core Workflows Protected by Tests
+
+| Workflow | Unit Tests | E2E Tests | Status |
+|----------|-----------|-----------|--------|
+| **1. User Authentication** | 6 auth tests | E2E login flow | ✅ Protected |
```
Comment (hmhawn) [inline docs/beta/week12-quality.md]: the E2E tests do not exercise real Google OAuth login. 

The Playwright setup uses:
x-test-user-id
/auth/user
test-mode auth helper

The doc should say something like:
“authenticated UI state via test-mode auth”
not
“E2E login flow”

**Review by hmhawn — CHANGES_REQUESTED** [review]
This is moving in the right direction, and the two new E2E tests are relevant to Week 12. I’m not ready to approve yet because docs/beta/week12-quality.md still misses a required link to a recent passing CI run, and a few statements in the report don’t match the current repo/CI setup. In particular, the E2E job is still continue-on-error: true, so the doc shouldn’t say CI fully enforces all tests before merge, and the Playwright suite uses test-mode auth rather than a real E2E login flow. Also, the new toggle test would be stronger if it saved the trip, reloaded it, and then verified the packed/unpacked states persisted instead of only checking checkbox state before save.
